recipe/meta.yaml

```diff
@@ -1,39 +1,53 @@
-{% set version = "0.11.7" %}
+{% set version = "0.11.10" %}
 
 package:
   name: matplotlib-venn
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/m/matplotlib-venn/matplotlib-venn-{{ version }}.tar.gz
-  sha256: 8d8b3505594313099328c9f177133dd800a28f3dd68e6800a78250c0cccce317
+  url: https://github.com/konstantint/matplotlib-venn/archive/refs/tags/{{ version }}.tar.gz
+  sha256: acb254f6fc0c8eb4e49df6655b316903d6babe2e9c0049636c2426a3a861cfbc
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-deps --no-build-isolation
+  skip: True  # [py<36]
```

Comment:
There's a `noarch` line and you have two `--no-deps`

Reply:
good catch, I've added suggestions